pkgs/by-name/wh/when/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, perl
+, installShellFiles
+, fetchFromBitbucket
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "when";
+  version = "1.1.46";
+
+  src = fetchFromBitbucket {
+    owner = "ben-crowell";
+    repo = "when";
+    rev = "a57ccbdf1d30e28a43a4bc5c68c37c171bb649ab";
+    hash = "sha256-uNIU4tsoJBBPj+2rYVvXl+D5f/GB80ox6s2MgxjuJa4=";
+  };
+
+  buildInputs = [ perl ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 when $out/bin/when
+    pod2man $src/when when.1
```

Comment:
This line should be in the `postBuild`.

Reply:
Why?

Reply:
I would say,  because its "building" man page

Reply:
Exact.

installPhase is to install artifacts generated from buildPhase, not to build thrngs.